packages/bpk-component-rating/src/common-types.js

```diff
@@ -18,6 +18,9 @@
 
 /* @flow */
 
-const RATING_SIZES = { sm: 'sm', base: 'base', lg: 'lg' };
+export const RATING_SIZES = { sm: 'sm', base: 'base', lg: 'lg' };
 
-export default RATING_SIZES;
+export const RATING_TYPE = {
```

Comment:
Can it be plural (`RATING_TYPES`) to match our convention of using plural elsewhere? 😇